core/pom.xml

```diff
@@ -33,6 +33,10 @@
   <name>Spark Project Core</name>
   <url>http://spark.apache.org/</url>
   <dependencies>
+    <dependency>
+      <groupId>com.thoughtworks.paranamer</groupId>
```

Comment:
do we need to include this in license? looks like the project has non-standard license text

Reply:
This is not a new dependency.
- https://github.com/apache/spark/blob/master/pom.xml#L1948-L1952

The license is here.
- https://github.com/apache/spark/blob/master/LICENSE-binary#L414